store/pg_engine.go

```diff
@@ -105,7 +105,7 @@ func (db *DB) Open(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
-	databaseName := db.connCfg.Username
+	databaseName := db.connCfg.Database
```

Comment:
@tianzhou I think that the failure is because of this line, that in other codepath `.Database` isn't set. Which follow up PR do you fix this issue?

Reply:
Right, this is related to how we derive and prepare the DB to be connected. I am working on a PR to fix that https://github.com/bytebase/bytebase/pull/1063.

I also created a doc draft to describe the enduser behavior: https://github.com/bytebase/bytebase.com/pull/70. PTAL.

Also since we are preparing a release this week and the overall feature is not ready, so we are going to turn off --pg for now. Will send out a separate PR later.

Reply:
> Also since we are preparing a release this week and the overall feature is not ready, so we are going to turn off --pg for now. Will send out a separate PR later.

OK. You can ping me on the next develop cycle and I'll try to push the feature to ready.